palindrome.py

```diff
@@ -0,0 +1,27 @@
+import re
+string = input("Enter your Text: ")
+string = re.sub(r'[^A-Za-z]',"",string)
+string = string.lower()
+print(string)
+def rec(inp):
+  inp = inp[:]
+
+  if len(inp)>2:
+      if inp[0]==inp[-1]:
+          return rec(inp[1:-1])
+      else:
+          return False
+
+  else:
+      if(len(inp)==1):
+          return True
+      else:
+          if inp[0]==inp[-1]:
+              return True
+          else:
+              return False
```

Comment:
A cool way to write less code: you could just return the result of `inp[0] == inp[-1]` instead of having an if statement here.